cmake/BLTPrivateMacros.cmake

```diff
@@ -599,8 +602,12 @@ macro(blt_split_source_list_by_language)
             if (DEFINED arg_Fortran_LIST)
                 list(APPEND ${arg_Fortran_LIST} ${_file})
             endif()
+        elseif(${_ext_lower} IN_LIST BLT_Python_FILE_EXTS)
+            if (DEFINED arg_Python_LIST)
+                list(APPEND ${arg_Python_LIST} ${_file})
+            endif()
         else()
-            message(FATAL_ERROR "blt_split_source_list_by_language given source file with unknown file extension. Add the missing extension to the corresponding list (BLT_C_FILE_EXTS or BLT_Fortran_FILE_EXTS).\n Unknown file: ${_file}")
+            message(FATAL_ERROR "blt_split_source_list_by_language given source file with unknown file extension. Add the missing extension to the corresponding list (BLT_C_FILE_EXTS, BLT_Fortran_FILE_EXTS, or BLT_Python_EXTS).\n Unknown file: ${_file}")
```

Comment:
Typo: BLT_Python_FILE_EXTS

Reply:
Good catch! Fixed in [8dfc85debc](https://github.com/LLNL/blt/pull/412/commits/8dfc85debcb3825cb3b85b2fe2e917dedfa954f3).